diff --git a/node_modules/wtf_wikipedia/types/index.d.ts b/node_modules/wtf_wikipedia/types/index.d.ts
index 8826379..2b9f0b8 100644
--- a/node_modules/wtf_wikipedia/types/index.d.ts
+++ b/node_modules/wtf_wikipedia/types/index.d.ts
@@ -1,4 +1,4 @@
-declare class Document {
+export declare class Document {
   categories(clue?: number): string[]
   category(clue?: number): string | null
   citations(clue?: number): Reference[]
@@ -49,7 +49,7 @@ declare class Document {
   wikitext(): string
 }
 
-declare class Section {
+export declare class Section {
   children(clue?: string | number): Section | Section[] | null
   citations: () => object | object[]
   coordinates(): object | object[]
@@ -182,7 +182,7 @@ declare class Sentence {
   wikitext(): string
 }
 
-export = wtf
+export default wtf
 
 type fetchDefaults = {
   path?: string | undefined;
