diff --git a/node_modules/wtf_wikipedia/types/index.d.ts b/node_modules/wtf_wikipedia/types/index.d.ts
index 8826379..7876209 100644
--- a/node_modules/wtf_wikipedia/types/index.d.ts
+++ b/node_modules/wtf_wikipedia/types/index.d.ts
@@ -1,4 +1,4 @@
-declare class Document {
+export declare class Document {
   categories(clue?: number): string[]
   category(clue?: number): string | null
   citations(clue?: number): Reference[]
@@ -49,11 +49,15 @@ declare class Document {
   wikitext(): string
 }
 
-declare class Section {
+export declare class Section {
   children(clue?: string | number): Section | Section[] | null
   citations: () => object | object[]
   coordinates(): object | object[]
   depth(): number
+  /**
+   * Added by the wtf-plugin-html
+   */
+  html(options: object): string
   images(): Image | Image[]
   indentation(): number
   index(): number | null
@@ -182,7 +186,7 @@ declare class Sentence {
   wikitext(): string
 }
 
-export = wtf
+export default wtf
 
 type fetchDefaults = {
   path?: string | undefined;
